db/migrate/20160216004441_add_name_to_games.rb

```diff
@@ -0,0 +1,5 @@
+class AddNameToGames < ActiveRecord::Migration
```

Comment:
When I merged Andrew's pull request, I forgot to include this migration in the commit. The schema knows about the game's name however. If it gives you problems do a rake db:drop, then rake db:migrate to fix.